git/refs/log.py

```diff
@@ -84,12 +84,12 @@ def message(self):
         return self[4]
 
     @classmethod
-    def new(cls, oldhexsha, newhexsha, actor, time, tz_offset, message):
+    def new(cls, oldhexsha, newhexsha, actor, time_, tz_offset, message):
```

Comment:
Is changing the `time` parameter name truly required? It would break people using the _keyword_ way of calling the method. If possible, I would prefer to keep it as is and silence deep source in other ways.

Reply:
Ok, I will revert it to make it silent,  It was not a critical issue.